azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageTypeServlet.java

```diff
@@ -74,10 +81,18 @@ protected void handleGet(
       throws ServletException, IOException {
     try {
       final Map<String, String> templateVariableToValue = new HashMap<>();
-      if (IMAGE_TYPE_WITH_ID_URI_TEMPLATE.match(req.getRequestURI(), templateVariableToValue)) {
-        // TODO: Implementation will be provided in the future PR
-      } else if (GET_IMAGE_TYPE_URI.equals(req.getRequestURI())) {
-        // TODO: Implementation will be provided in the future PR
+      ImageTypeDTO imageTypeDTO;
+      if (IMAGE_TYPE_WITH_NAME_URI_TEMPLATE.match(req.getRequestURI(), templateVariableToValue)) {
```

Comment:
The comment in the class indicates that the endpoint `GET /imageTypes?&imageType=?` is supported could you point to its implementation?

Reply:
It is not currently supported. I will remove it. Thank you.